<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="AED">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>755</width>
      <height>729</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/AED_Image.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="power_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>580</y>
      <width>69</width>
      <height>55</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: transparent;
	background: none;
	border:none;
	background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="AED_Images.qrc">
      <normaloff>:/Images/Power_Button.png</normaloff>:/Images/Power_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shock_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>80</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: transparent;
	background: none;
	border:none;
	background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="AED_Images.qrc">
      <normaloff>:/Images/Shock_Button.png</normaloff>:/Images/Shock_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="led_indicator_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>273</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/LED_Indicator.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="led_indicator_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>170</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/LED_Indicator.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="led_indicator_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>320</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/LED_Indicator.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="led_indicator_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>420</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/LED_Indicator.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="led_indicator_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>430</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/LED_Indicator.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="self_test_failed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/self_test_off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="self_test_succeeded">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/self_test_on.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lcd_screen">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>215</y>
      <width>165</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgrey</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shock_label">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>220</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
     </font>
    </property>
    <property name="text">
     <string>SHOCKS:</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="display_message">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Liberation Sans'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detach_pads">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect Pads</string>
    </property>
   </widget>
   <widget class="QLabel" name="heart_rhythm_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Heart Rhythm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VF_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ventricular Fibrillation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VT_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ventricular Tachycardia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PEA_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>310</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PEA</string>
    </property>
   </widget>
   <widget class="QLabel" name="CPR_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CPR Strength</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asystole_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Asystole</string>
    </property>
   </widget>
   <widget class="QSlider" name="compression_strength">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>430</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_compression_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Compression</string>
    </property>
   </widget>
   <widget class="QLabel" name="shock_count">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>220</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="elapsed_time">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="led_indicator_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="AED_Images.qrc">:/Images/LED_Indicator.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="defib_pads_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach and Detach Pads</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defib_pads_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach Defib Pads</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="battery_percentage_bar">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	text-align: center;
}

QProgressBar::chunk{
	background-color: rgb(87, 227, 137);
}</string>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="heart_rhythm_label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery Percentage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replace_battery_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace Battery</string>
    </property>
   </widget>
   <widget class="QLabel" name="heart_rhythm_graph_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Heart Rhythm Graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="heart_rhythm_graph">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="compression_indicator">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>240</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	text-align: center;
	background-color: rgb(192, 191, 188);
}

QProgressBar::chunk{
	background-color: rgb(205, 171, 143);
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="AED_Images.qrc"/>
 </resources>
 <connections/>
</ui>
